client/chain/chain.go

```diff
@@ -272,7 +272,11 @@ func NewChainClient(
 	}
 
 	// create file if not exist
-	os.OpenFile(defaultChainCookieName, os.O_RDONLY|os.O_CREATE, 0666)
+	_, err = os.OpenFile(defaultChainCookieName, os.O_RDONLY|os.O_CREATE, 0666)
+	if err != nil {
+		cc.logger.Errorln(err)
+	}
+	defer cookie_file.Close()
```

Comment:
`cookie_file` doesn't exist, plus OpenFile without storing result won't result in memory leak (it will be GCed)

Reply:
Thank you for the review @vinhphuctadang. Even when closing the file is not strictly required, isn't it a good practice to close all the files that are opened in the code? Besides it is just 4 extra lines that could never cause an issue. I still thin we should keep the change

Reply:
Yeah, I agree it's good practice, but please check the `cookie_file` var, it doesn't exist
<img width="371" alt="image" src="https://github.com/InjectiveLabs/sdk-go/assets/30641530/953e6485-1906-4326-b5d1-bdabd1db18e4">

Reply:
Thank you for that. It is fixed now.